server/api/user.go

```diff
@@ -140,6 +140,26 @@ func GetRepos(c *gin.Context) {
 		return
 	}
 
+	repoIDs := make([]int64, len(activeRepos))
+	for i, repo := range activeRepos {
+		repoIDs[i] = repo.ID
+	}
+
+	pipelines, err := _store.GetRepoLatestPipelines(repoIDs)
```

Comment:
What do you think about a query parameter to include the latest pipeline and leaving it out by default?

Reply:
Do you worry about the performance or something else?

Reply:
I did not find any case where it would be called without needing this, so I don't see much point in this currently